### ISSUE TITLE
[FEATURE] We need to be able to call modal windows from the sidebar menu described in the EventServiceProvider

### ISSUE THREAD

**Issue by vitkuz573** [jeroennoten/Laravel-AdminLTE#1021]
### Is your request related to a problem? Please describe.
No way to call the modal window from the sidebar

### Describe the solution you'd like
We need to be able to call modal windows from the sidebar menu described in the EventServiceProvider

### Describe alternatives you've considered
Hadn't considered

**Comment by Shidersz** [comment]
Currently, you may put a link on the left sidebar to open a modal that is defined globally using next workaround:

1. You should define a base layout for your application and put your modal there to be available globally, see issue #925 to check for an example on how to create a base layout. An example for your base layout may be:
   ```blade
   {{-- resource/views/layout.blade.php --}}

   @extends('adminlte::page')

   {{-- You may add other layout customizations here... --}}

   {{-- Global Modal --}}
   <x-adminlte-modal id="exampleModal" title="Minimal"/>
   ```
2. Then, you may define next item on your sidebar menu to toggle the modal:
   ```php
   [
       'text'   => 'Open Modal',
        'url'    => '',
        'data'   => [
            'toggle' => 'modal',
            'target' => '#exampleModal',
        ]
   ]
   ```

**Comment by vitkuz573** [comment]
Thank you, the question is closed.
